src/main/haskell/kore/test/prover/Test/Kore/MatchingLogic/ProofSystem/OnePlusOne/ProofSystem.hs

```diff
@@ -99,7 +98,8 @@ transformRule _sort label var term hypothesis rule = case rule of
     OrIntroR t1 t2 -> OrIntroR <$> term t1 <*> term t2
 
 -- | Lens focusing on the terms within a Rule.
-ruleTerms :: (Applicative f)
+ruleTerms :: forall (f :: * -> *) (sort :: *) termA termB hyp label var.
```

Comment:
Why was an explicit forall added here?

Reply:
I needed to add an explicit kind signature because of enabling `PolyKinds` by default.